src/components/DoctorCard/PageInfo/WebsiteLinks.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import DoctorLinks from '../Shared/DoctorLinks';
+
+function withDoctorLinks(Component) {
+  const WebsiteLinks = function WebsiteLinks({ website }) {
+    const websites = website
+      ?.split(',')
+      .map(w => {
+        const url = w?.trim();
+        if (url.startsWith('http')) {
+          return new URL(url);
+        }
+        return new URL(`http://${url}`);
+      })
+      .filter(w => Boolean(w));
```

Comment:
This needs to be more robust.
What if someone makes a mistake and website = 'doctor.com, '  => url would be ''

Reply:
Should refurn undefined (or null) if !url